Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3
```

Comment:
With this tag, Docker will always pull the latest stable python version. This is currently 3.6.1. If desired, we can specify a version like 2.7 or 3.5.